# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/moz.build
# Commit: 2a040986845d
# Full Hash: 2a040986845da97f1eaf87c33136d9168817606a
# Author: Neil Deakin <neil@mozilla.com>
# Date: 2015-04-13 10:00:52
# Regressor Bug: 1071562
# File Overlap Count: 1
# Description:
#   Bug 1071562, e10s, support non-text types in clipboard (html, images, etc), r=smaug
# ==============================================================================

diff -r 8975ed86f399 -r 2a040986845d dom/base/moz.build
--- a/dom/base/moz.build	Fri Apr 10 09:43:38 2015 -0700
+++ b/dom/base/moz.build	Fri Apr 10 12:51:06 2015 -0400
@@ -415,6 +415,7 @@
     '/dom/xml',
     '/dom/xslt/xpath',
     '/dom/xul',
+    '/gfx/2d',
     '/image/src',
     '/js/xpconnect/src',
     '/js/xpconnect/wrappers',